packages/xo-server/src/api/vm.mjs

```diff
@@ -173,7 +235,7 @@ export const create = defer(async function ($defer, params) {
   }
 
   if (params.bootAfterCreate) {
-    ignoreErrors.call(xapi.startVm(vm._xapiId))
+    startVmAndDestroyCloudConfigVdi(xapi, vm, cloudConfigVdiUuid, params)
```

Comment:
I think the ingoreError should be for the full function . That way the error won't bubble to the node process if they occur in this phase of the vm creation 
```suggestion
    ignoreErrors.call(startVmAndDestroyCloudConfigVdi(xapi, vm, cloudConfigVdiUuid, params))
```